consensus/src/quorum/verifiers.rs

```diff
@@ -205,12 +205,13 @@ pub async fn verify_votes(
         return Err(Error::VoteSetTooSmall(cfg.step));
     }
 
-    // aggregate public keys
-    let apk = sub_committee.aggregate_pks()?;
-
-    // verify signatures
-    verify_step_signature(cfg.round, cfg.step, block_hash, apk, signature)?;
+    if bitset > 0 {
```

Comment:
I'm not sure about this check here. Isn't it unsafe allowing empty signatures to pass verifiers?

Reply:
For usual checks, if the bitset it set then we'll shortcut the error due to not enough step_votes

Reply:
Added comments to explain